examples/examples.py

```diff
@@ -411,9 +411,16 @@ def stft_plot(signal, sample_rate=SAMPLE_RATE):
 # ## Torch examples
 
 
+# +
 import torch
 from ddspdrum.torchmodule import TorchADSR
 
+if torch.cuda.is_available():
+    device = "cuda:0"
```

Comment:
I'm not 100% sure about this, I think if you use "cuda" then it uses all your GPUs. We have an issue later to test it: https://github.com/turian/torchsynth/issues/71 so this is fine for now